From 1ddb7c1f46fadf28a71f2f8bd7733204947f754e Mon Sep 17 00:00:00 2001
From: celadon <celadon@intel.com>
Date: Mon, 3 Feb 2025 08:23:14 +0000
Subject: [PATCH] Changing the enum values

These enum values are changed since only value till
31 are defined and more than that is not allowed.

Tests Done: Build proceed

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 cros_gralloc/i915_private_android_types.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/cros_gralloc/i915_private_android_types.h b/cros_gralloc/i915_private_android_types.h
index 7ef8df2..38d750f 100644
--- a/cros_gralloc/i915_private_android_types.h
+++ b/cros_gralloc/i915_private_android_types.h
@@ -59,11 +59,11 @@ enum { HAL_PIXEL_FORMAT_NV12_Y_TILED_INTEL = 0x100,
        HAL_PIXEL_FORMAT_YUV420PackedSemiPlanar_Tiled_INTEL = 0x7FA00F00,
 };
 
-enum { GRALLOC1_FUNCTION_SET_MODIFIER = 101,
-       GRALLOC1_FUNCTION_GET_BYTE_STRIDE = 102,
-       GRALLOC1_FUNCTION_GET_PRIME = 103,
+enum { GRALLOC1_FUNCTION_SET_MODIFIER = 28,
+       GRALLOC1_FUNCTION_GET_BYTE_STRIDE = 29,
+       GRALLOC1_FUNCTION_GET_PRIME = 30,
 
-       GRALLOC1_LAST_CUSTOM = 500 };
+       GRALLOC1_LAST_CUSTOM = 31 };
 
 typedef int32_t /*gralloc1_error_t*/ (*GRALLOC1_PFN_SET_MODIFIER)(
     gralloc1_device_t *device, gralloc1_buffer_descriptor_t descriptor, uint64_t modifier);
-- 
2.34.1

